app/vep/vep_resources.py

```diff
@@ -210,3 +212,38 @@ async def fetch_results(request: Request, submission_id: str, page: int, per_pag
     except Exception as e:
         logging.error(e)
         return response_error_handler(result={"status": 500})
+
+@router.get("/form_config/{genome_id}", name="get_form_config")
+async def get_form_config(request: Request, genome_id: str):
+    try:
+        attributes = await get_genome_metadata(genome_id)
+        annotation_provider_name = attributes.get("genebuild.provider_name")
+        annotation_version = attributes.get("genebuild.display_version")
+        last_updated_date = attributes.get("genebuild.last_geneset_update")
+
+        transcript_set=TranscriptSet(
+            label=f"{annotation_provider_name} {annotation_version or last_updated_date}",
+        )
+
+        form_config = FormConfig(
+            transcript_set = transcript_set
+        )
+        return form_config
+
+    except HTTPError as http_error:
+        if http_error.response.status_code in [403, 400]:
```

Comment:
`403, 400` is probably incorrect?

What should happen after the exception is caught? Probably just log the error and send the same response to client?